applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -256,6 +256,10 @@ namespace Kratos
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, Vector, EFFECTIVE_COMPRESSION_STRESS_VECTOR)
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, EXPONENTIAL_SATURATION_YIELD_STRESS)
     KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, ACCUMULATED_PLASTIC_STRAIN)
+    KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, Vector, HIGH_CYCLE_FATIGUE_PARAMETERS)
+    KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, FATIGUE_REDUCTION_FACTOR)
+    KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, NUMBER_OF_CYCLES)
+    KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, WOHLER_STRESS)
```

Comment:
the german name is Wöhler, the correct translation would be Woehler
Isn't it like this in the english literature?

Reply:
I have used the name that appears in literature when building the CL and it sounds more natural to me to keep in this way

Reply:
ok